features/steps/standby_cluster.py

```diff
@@ -37,12 +42,18 @@ def start_patroni_stanby_cluster(context, name, cluster_name, name2):
         "scope": cluster_name,
```

Comment:
Would this be the right time to fix the function name `start_patroni_stanby_cluster` (a missing d)?